src/reader.cpp

```diff
@@ -2044,6 +2051,8 @@ void reader_set_test_function(parser_test_error_bits_t (*f)(const wchar_t *)) {
 
 void reader_set_exit_on_interrupt(bool i) { data->exit_on_interrupt = i; }
 
+void reader_set_silent_status(bool f) { data->silent = f; }
```

Comment:
It was meant to be the abbreviation of flag, but if more meaning is needed i can write the whole word